doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,106 @@ a single confidential command to let the redirection also apply for 'set -x'.
 See the description in /usr/share/rear/conf/default.conf
 how to set variables for secret values in a confidential way.
 
+== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM
+
+There are two options how to boot the rescue environment on IBM Z (s390x) under
+z/VM created using the `OUTPUT=RAMDISK` setting.
```

Comment:
Other scenarios in this document use a config file snippet. Like
> If you want a bootable USB device with a (tar) backup to USB as well, you would use:
> ```
> BACKUP=NETFS
> OUTPUT=USB
> USB_DEVICE=/dev/disk/by-label/REAR-000
> ```

I think you should add this kind of snippet as well.

Reply:
Added in 873884f.